### PR TITLE
Fix indentations as per eslint 4.0.0

### PR DESCRIPTION
**Pull request by pelinKuran**
Change-Id: Idf952d4ba52c291117050ca9f5b0f021690c5b81

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [X] Code is properly formatted
- [X] All commits have Change-Id
- [ ] I have run tests with `make check`
- [X] I have issued `make run` and manually verified that everything looks okay
- [X] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: loleaflet/src/layer/tile/CanvasTileLayer.js [hunk 1]
```diff
@@ -324,11 +324,11 @@ L.CanvasTilePainter = L.Class.extend({
 
 					var key = coords.key();
 					var tile = this._layer._tiles[key];
-//					var invalid = tile && tile._invalidCount && tile._invalidCount > 0;
+					//var invalid = tile && tile._invalidCount && tile._invalidCount > 0;
 					if (tile && tile.loaded) {
 						this.paint(tile, ctx);
 					}
-/*					else
+					/*					else
```
Comment (mrkara) [inline loleaflet/src/layer/tile/CanvasTileLayer.js:331]: This doesn't look right.

**Review by mrkara — CHANGES_REQUESTED** [review]

File: loleaflet/src/layer/tile/CanvasTileLayer.js (hunk 1 above)
Comment (pelinKuran) [reply loleaflet/src/layer/tile/CanvasTileLayer.js:331]: it should be like 

`/*else `

right?

Comment (mrkara) [reply loleaflet/src/layer/tile/CanvasTileLayer.js]: Right. Thanks!

**Review by mrkara — APPROVED** [review]

**Comment by mrkara** [comment]
@pelinKuran Thank you! :)

**Pull request merged**